charts/trow/templates/ingress.yaml

```diff
@@ -35,8 +35,8 @@ spec:
         {{- range .paths }}
           - path: {{ . }}
             backend:
-              serviceName: {{ $fullName }}
-              servicePort: {{ $svcPort }}
+              service:
+                name: {{ $fullName }}
```

Comment:
I can't remember now but I do remember making a decision to remove the port. I think it might have not worked if I left the port in as $svsPort wasn't valid? But I can't remember how I tested that file anymore unfortunately sorry. 
I've added port: {{$svsPort}} back in and quick install and helm and kustomize seem to work fine.